docs/how-to/revert-from-rt-kernel.md

```diff
@@ -0,0 +1,121 @@
+# How to revert from real-time kernel to generic kernel
```

Comment:
File name doesn't make so much sense: change to switch-from-realtime-to-generic-kernel.md ?


Page title match the index.